lib/aullar/flair.moon

```diff
@@ -35,8 +35,21 @@ set_line_type_from_flair = (cr, flair) ->
       cr.dash = {6, 3}
 
 draw_ops = {
-  rectangle: (flair, x, y, width, height, cr) ->
+  custom: (flair, x, y, width, height, cr) ->
+    if flair.custom_draw and
+         pcall flair.custom_draw, flair, x, y, width, height, cr
+      return
+    else
+      -- Draw an intensely red "shit has gone bad"-cross instead
```

Comment:
@shalabhc - In the case of the flair being for cursor, it is nice that you still see where you're editing. Also I figure there's a much greater chance it's not skimped over, resulting in an editor with tons of "invisibly failing features" eating cycles (see that happening in many plugins in many editors else where). It seems human mentality is "see no evil - there is no evil", haha.

@shalabhc  Perhaps the code formatting was badly chosen: there's: `and` on line 39...